src/app.module.ts

```diff
@@ -47,6 +48,12 @@ const ENV = process.env.NODE_ENV;
         }),
         ScheduleModule.forRoot(),
         CacheModule.register({ isGlobal: true }),
+        BullModule.forRoot({
+            redis: {
+                host: 'localhost',
+                port: 6379,
```

Comment:
Does this need to be configurable for Heroku?

Reply:
Could be yes. Not sure if redis is local or external there